deploy/12_old_catalysts/09_setup_starter_pack.ts

```diff
@@ -0,0 +1,39 @@
+import {BigNumber} from 'ethers';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+// sand price is in Sand unit (Sand has 18 decimals)
+const starterPackPrices = [sandWei(0), sandWei(0), sandWei(0), sandWei(0)];
+const gemPrice = sandWei(0);
+
+function sandWei(amount: number) {
+  return BigNumber.from(amount).mul('1000000000000000000').toString();
+}
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments} = hre;
+  const {read, execute, catchUnknownSigner} = deployments;
+  const prices = await read('StarterPackV1', {}, 'getPrices');
+
+  const needsSetup =
+    prices.pricesAfterSwitch.some(
+      (price: BigNumber, i: number) => !price.eq(starterPackPrices[i])
+    ) || !prices.gemPriceAfterSwitch.eq(gemPrice);
+  if (needsSetup) {
+    console.log('Setting up new prices');
+    const admin = await read('StarterPackV1', {}, 'getAdmin');
+    await catchUnknownSigner(
+      execute(
+        'StarterPackV1',
+        {from: admin},
+        'setPrices',
+        starterPackPrices,
+        gemPrice
+      )
+    );
+  }
+};
+export default func;
+func.tags = ['StarterPackV1', 'StarterPackV1_setup'];
+func.dependencies = [];
+func.skip = async () => true;
```

Comment:
always skip ?

Reply:
The deploy always skips too
https://github.com/thesandboxgame/sandbox-smart-contracts/blob/9151009d6234a3832caf49c43f0c4d9d742c0e9b/deploy/12_old_catalysts/07_deploy_starter_pack.ts#L56

Reply:
I changed it to skipUnlessTest